hawkular-inventory-itest-parent/hawkular-inventory-itest-dist/pom.xml

```diff
@@ -0,0 +1,592 @@
+                      <parameters>
+                        <parameter>
+                          <name>kettle.build.type</name>
+                          <value>${kettle.build.type}</value>
+                        </parameter>
+                        <parameter>
+                          <name>uuid.hawkular.accounts.backend</name>
+                          <value>${uuid.hawkular.accounts.backend}</value>
+                        </parameter>
+                      </parameters>
+                    </transformationSet>
+                    <!-- standalone-itest.xml -->
+                    <transformationSet>
+                      <dir>${project.build.directory}/wildfly/${wildfly.dist.zip.root.dir}/standalone/configuration</dir>
+                      <stylesheet>src/main/resources/wildfly/patches/standalone.xsl</stylesheet>
+                      <fileMappers>
+                        <fileMapper implementation="org.codehaus.plexus.components.io.filemappers.RegExpFileMapper">
+                          <pattern>\.xml$</pattern>
+                          <replacement>-itest.xml</replacement>
+                        </fileMapper>
+                      </fileMappers>
+                      <includes>
+                        <include>standalone.xml</include>
+                      </includes>
+                      <outputDir>${assembly.dir}/${hawkular.dist.zip.root.dir}/standalone/configuration</outputDir>
+                      <parameters>
+                        <parameter>
+                          <name>kettle.build.type</name>
+                          <value>dev</value>
+                        </parameter>
+                        <parameter>
+                          <name>uuid.hawkular.accounts.backend</name>
+                          <value>${uuid.hawkular.accounts.backend}</value>
+                        </parameter>
+                      </parameters>
+                    </transformationSet>
+                  </transformationSets>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+
+          <plugin>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <configuration>
+              <finalName>${project.artifactId}-${project.version}</finalName>
+              <appendAssemblyId>false</appendAssemblyId>
+              <descriptors>
+                <descriptor>assembly.xml</descriptor>
+              </descriptors>
+            </configuration>
+            <executions>
+              <execution>
+                <id>distro-assembly</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>single</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+
+          <plugin>
+            <groupId>com.spotify</groupId>
+            <artifactId>docker-maven-plugin</artifactId>
```

Comment:
I'd say that `docker-maven-plugin` should not be here, as it should not be in agent-itest-dist